go/itcases/main_test.go

```diff
@@ -32,11 +32,12 @@ func TestMain(m *testing.M) {
 	goleak.VerifyTestMain(m)
 }
 
-func NewClient() *scopedb.Client {
+func NewClient(t testing.TB) *scopedb.Client {
 	endpoint := os.Getenv("SCOPEDB_ENDPOINT")
```

Comment:
I suggest you to enable usetesting linter via golangci-lint.

It will show you that you can use t.Getenv here